zrml/prediction-markets/src/benchmarks.rs

```diff
@@ -225,26 +225,13 @@ fn setup_resolve_common_scalar_after_dispute<T: Config>(
 
 benchmarks! {
     admin_destroy_disputed_market{
```

Comment:
You benchmark it in another PR? (after you corrected the benchmark for `on_resolution`), right?

Reply:
yes.